integ-test-promote-template.yml

```diff
@@ -426,3 +430,31 @@ jobs:
             grep "checking" buildlog.txt
             rm buildlog.txt
           displayName: Verify that correct tasks appear in buildlog.txt
+        - task: MathWorks.matlab-azure-devops-extension-dev.RunMATLABBuild.RunMATLABBuild@${{ version }}
+          displayName: Run build with task skipping
+          inputs:
+            tasks: deploy
+            build-options: -skip test
+        - bash: |
+            set -e
+            grep "building" buildlog.txt
+            ! grep "testing" buildlog.txt
+            grep "deploying" buildlog.txt
+            ! grep "checking" buildlog.txt
```

Comment:
Nitpick: This line probably isn't necessary (just mentioning here because the pipeline is getting looooong lol)